postmark/core.py

```diff
@@ -338,9 +345,13 @@ def _check_values(self):
             raise PMMailMissingValueException('Cannot send an e-mail without a sender (.sender field)')
         elif not self.__to:
             raise PMMailMissingValueException('Cannot send an e-mail without at least one recipient (.to field)')
-        elif not self.__subject:
+        elif bool(self.__template_id) != bool(self.__template_model):
```

Comment:
Had to think about what was happening here for a bit. I think this reads easier as:

```
elif (self.__template_id or self.__template_model) and not all([self.__template_id or self.__template_model]):
```